browser/src/control/ColorPicker.ts

```diff
@@ -378,11 +385,11 @@ L.ColorPicker = L.Class.extend({
 		}
 	},
 
-	_getSampleElement: function (index, type) {
+	_getSampleElement: function (index: number, type: ColorTypes) {
 		var sampleId;
-		if (type === L.ColorPicker.BASIC_COLOR) {
+		if (type === "BASIC_COLOR") {
 			sampleId = this._basicColorSampleIdTag + index;
-		} else if (type === L.ColorPicker.TINT) {
+		} else if (type === "TINT") {
```

Comment:
do we allow " now ?

Reply:
no, but I'm used to doing it and forgot to run the formatter - I'll add it to a hook or something